addons/mass_mailing/i18n/mass_mailing.pot

```diff
@@ -257,7 +257,8 @@ msgstr ""
 
 #. module: mass_mailing
 #: model:ir.model.constraint,message:mass_mailing.constraint_mailing_contact_list_rel_unique_contact_list
-msgid "A mailing contact cannot subscribe to the same mailing list multiple times."
+msgid ""
```

Comment:
Is it intended ?

Reply:
The auto generated file makes this part in two lines. I don't know if that changes something so I kept it like this

Reply:
@mart-e do we keep "void lines" ?

Reply:
It's handled by `polib`, not sure what makes him think a new line is a better but I try not to argue with external libs. #AllLinesAreBeautiful